Business.Queries/JobPositions/GetJobPositionByIdQueryHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Business.Queries.Dtos;
+using CCG.AspNetCore.Business.Interface;
+using DataModel;
+using Microsoft.EntityFrameworkCore;
+
+namespace Business.Queries.JobPositions
+{
+    public class GetJobPositionByIdQueryHandler
```

Comment:
Needs to implement IQueryHandler

Reply:
@alfred84-boum Hi Alfred, I saw you fixed some of the issues. You still need to merge changes from master into this branch before it can be completed.